API.md

```diff
@@ -0,0 +1,186 @@
+<!-- Generated by documentation.js. Update this documentation by updating the source code. -->
+
+### Table of Contents
+
+- [Test Waiter Manager][1]
+  - [register][2]
+    - [Parameters][3]
```

Comment:
Can we follow suite with @ember/test-helpers here and use the TOC depth option? The `Parameters` shenanigans in the table of contents drives me nuts!

Reply:
Oh yes, absolutely. I didn't know how to filter that.

Reply:
Fixed!